lisp/ess-r-mode.el

```diff
@@ -870,26 +855,6 @@ returned."
       (setq date (match-string 2 ver-string)))
     (cons date rver)))
 
-(defun ess-current-R-version ()
-  "Get the version of R currently running in the ESS buffer as a string."
-  (ess-make-buffer-current)
-  (car (ess-get-words-from-vector "as.character(.ess.Rversion)\n")))
-
-(defun ess-current-R-at-least (version)
-  "Is the version of R (in the ESS buffer) at least (\">=\") VERSION ?
```

Comment:
Why are these functions removed? These are public functions designed to retrieve versions from R. They might be used by people's code.

Reply:
I thought that they were probably used at one point by ESS but weren't any longer. If they're meant to be external, I can bring them back easily enough. Just let me know if you want me to (or feel free to do so yourself).

Reply:
I would say we keep the first two. At some point even we might need them and that would mean re-inventing the code.